views/openJobRoleList.html

```diff
@@ -4,26 +4,26 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     {% include "links.html" %}
-    <title>Open Job Role List</title>
+    <title id="">Open Job Role List</title>
```

Comment:
Why empty?

Reply:
i dont think title needs an id as it's the tab of the page so wouldn't be used

Reply:
then remove the id element? better than having a blank one

Reply:
can someone update the ticket to document that as an aside, the id's were added under it